config/initializers/xapi_lrs.rb

```diff
@@ -0,0 +1,5 @@
+$remote_lrs = Xapi.create_remote_lrs(
```

Comment:
instead of global I will add it to Rails.configuration - something like this:

```ruby
Rails.configuration.to_prepare do
  Rails.configuration.remote_lrs = Xapi.create_remote_lrs(
    end_point: ENV['LRS_ENDPOINT'],
    user_name: ENV['LRS_USERNAME'],
    password: ENV['LRS_PASSWORD']
  )
end
```